tests/unit/admin/test-class-wc-rest-payments-readers-controller.php

```diff
@@ -291,6 +291,118 @@ public function test_generate_print_receipt() {
 		$this->assertSame( 200, $response->status );
 	}
 
+	public function test_preview_print_receipt() {
+		$this->mock_wcpay_gateway
+			->expects( $this->never() )
+			->method( 'get_option' );
+
+		$this->mock_receipts_service
+			->expects( $this->once() )
+			->method( 'get_receipt_markup' )
+			->with(
+				[
+					'business_name' => 'Test',
+					'support_info'  => [
+						'phone'   => '424242',
+						'email'   => 'some@example.com',
+						'address' => [
+							'line1'       => 'line1',
+							'line2'       => 'line2',
+							'city'        => 'city',
+							'state'       => 'state',
+							'postal_code' => 'postal_code',
+							'country'     => 'country',
+						],
+					],
+				],
+				$this->controller->create_print_preview_receipt_order_data(),
+				$this->controller->create_print_preview_receipt_charge_data()
+			)
+			->willReturn( '' );
```

Comment:
a sample response (could be also a just a random text) would be useful to avoid a possible error when no text is returned.

Reply:
Good point thanks

Reply:
Done